bluebottle/categories/models.py

```diff
@@ -43,3 +36,20 @@ def __unicode__(self):
 
     def get_absolute_url(self):
         return 'https://{}/projects/?category={}'.format(properties.tenant.domain_url, self.slug)
+
+
+class CategoryContent(models.Model):
+    category = models.ForeignKey(Category, related_name='contents')
+    title = models.CharField(_('title'), max_length=60)
+    description = models.TextField(_("description"), max_length=190)
+    image = ImageField(_("image"), max_length=255, blank=True, null=True, upload_to='categories/content/',
+                       help_text=_("Category content image"))
+    video_url = models.URLField(max_length=100, blank=True, default='')
+    link = models.URLField(_('Link url'), blank=True)
```

Comment:
I think you also need a title to the link, so you can say "View all projects" or "Read more". Maybe check with Rolf?